src/Bicep.Core/Semantics/Namespaces/ThirdPartyNamespaceType.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Immutable;
+using Bicep.Core.TypeSystem;
+using Bicep.Core.TypeSystem.Providers;
+using Bicep.Core.TypeSystem.Providers.ThirdParty;
+using Bicep.Core.TypeSystem.Types;
+
+namespace Bicep.Core.Semantics.Namespaces
+{
+    public static class ThirdPartyNamespaceType
+    {
+        public static NamespaceSettings Settings { get; } = new(
+            IsSingleton: true,
+            BicepProviderName: string.Empty,
+            ConfigurationType: null,
+            ArmTemplateProviderName: string.Empty,
+            ArmTemplateProviderVersion: string.Empty);
+
+        public static NamespaceType Create(string name, string aliasName, IResourceTypeProvider resourceTypeProvider)
+        {
+            return new NamespaceType(
+                aliasName,
+                new NamespaceSettings(
+                    IsSingleton: true,
+                    BicepProviderName: name,
+                    ConfigurationType: null,
+                    ArmTemplateProviderName: name[0].ToString().ToUpper() + name.Substring(1),
```

Comment:
This may not be necessary, but I noticed the pattern for other ArmTemplateProviderName was uppercasing the first letter. 
Ex. AzureResourceManager

Did the same here.

Reply:
I think ultimately it would make sense for this information to come from the types package, but this feels out of scope for this particular PR. For now I would recommend just leaving as is (without changing casing).